container-templates/docker-compose/.devcontainer/devcontainer.json

```diff
@@ -1,11 +1,25 @@
 {
-    // "See https://aka.ms/vscode-remote/devcontainer.json for format details."
-    "name": "Dev Container Definition Template - Docker Compose",
-    "dockerComposeFile": "docker-compose.yml",
-    "service": "your-service-name-here",
-    "workspaceFolder": "/workspace",
-    "shutdownAction": "stopCompose",
-    "extensions": [
-        "mutantdino.resourcemonitor"
-    ]
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
+	"name": "Dev Container Definition Template - Docker Compose",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	"dockerComposeFile": "docker-compose.yml",
+
+	// The 'service' property is the name of the service for the container that VS Code should
+	// use. Update this value and .devcontainer/docker-compose.yml to the real service name.
+	"service": "your-service-name-here",
+
+	// The optional 'workspaceFolder' property is the path VS Code should open by default when
+	// connected. This is typically a volume mount in .devcontainer/docker-compose.yml
+	"workspaceFolder": "/workspace",
+
+	// Add any extensions you want auto-installed here.
+	"extensions": [],
+
+	// Update the value of this line to 'none' if you want to keep your containers running after VS Code shuts down.
+	"shutdownAction": "stopCompose",
+
+	// Uncomment the next line to run commands after the container is created.
+	// "postCreateCommand": ["uname -a"]
```

Comment:
@Chuxel `["uname -a"]` won't work. You can use `"uname -a"` or `["uname", "-a"]`. This is similar to `CMD`, `RUN` and others in the Dockerfile. The single string will be parsed by a shell and then executed, the array form is executed directly, without a shell.